### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by RasPhilCo**

### CODE REVIEW

**Comment by RasPhilCo** [comment]
@RodEsp if you have a moment soon, do you mind giving this README update a once over?

File: README.md [hunk 1]
```diff
@@ -11,9 +11,92 @@ multi-repo syncronization &amp; management tool
...
+
+* [file-does-not-exist](#type-file-does-not-exist)
+* [file-is-exact](#type-file-is-exact)
+
+#### type: file-does-not-exist
+
+Property | Example | Description
+--- | --- | ---
+description | `remove old LICENSE` | describe what is transpiring, which will become the commit message
+if | `test -e "./new/LISCENSE.txt"` | assertion only runs if this bash expression returns successfully
```
Comment (RodEsp) [inline README.md:88]: Do `if`s have to be bash expressions?

File: README.md [hunk 2]
```diff
@@ -11,9 +11,92 @@ multi-repo syncronization &amp; management tool
...
+target_relative_filepath (required) | `./old/LICENSE.txt` | relative filepath to target file (relative to repo root)
+
+#### type: file-is-exact
+
+Property | Example | Description
+--- | --- | ---
+description | `add LICENSE` | describe what is transpiring, which will become the commit message
+if | `foo` | assertion only runs if this bash expression (run at the repo root) returns successfully
```
Comment (RodEsp) [inline README.md:96]: Do `if`s have to be bash expressions?

**Review by RodEsp — CHANGES_REQUESTED** [review]
Overall looks good to me but one of the toc links is broken.

File: README.md (hunk 2 above)
Comment (RasPhilCo) [reply README.md:96]: Currently, yes. Could make it run a shell script.

File: README.md (hunk 1 above)
Comment (RasPhilCo) [reply README.md:88]: Currently, yes. Could make it run a shell script.

**Pull request merged**